aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SubscriptionProcessor.java

```diff
@@ -115,14 +125,13 @@ void startSubscriptions() {
             ));
             latch.await(SUBSCRIPTION_START_TIMEOUT_MS, TimeUnit.MILLISECONDS);
         })
-        .doOnError(exception -> {
-            LOG.warn("An error occurred with one of the subscriptions to the remote DataStore for " +
-                "model " + clazz.getSimpleName() + " " + subscriptionType.name(),
-                exception.getCause());
-        })
-        .onErrorResumeNext(next -> {
-            next.onComplete();
-        })
+        .doOnError(subscriptionError ->
+            LOG.warn(String.format(Locale.US,
```

Comment:
Formatting seems a little off.

Reply:
This is actually intentional, I was trying to have it like:

```
wrapped(crap(i(dont(really(care(about(
    "A clear log message on its own line! %s %s",
    withSupporting, dataToBePluggedIn
))))));
```